docs/update_benchmark_tables.py

```diff
@@ -232,14 +257,16 @@ def main(data_wrapper_api_token: str):
     for subset, result in results.items():
         langs = BENCHMARKS[subset]
 
-        table = convert_to_table(result, langs)
+        raw_table = convert_to_table(result, langs)
+        table = raw_table.drop(columns=["Model name"])
```

Comment:
I'm not sure I understand the intention here. Why are we dropping the column exactly?

Reply:
Otherwise, it will be shown in the table (and we don't want that, we want the model name with the link)